tests/unit/kvmagent/test_network_plugin_handlers.py

```
@@ -0,0 +1,1567 @@
+                    }
+                }
+            }
+        })
+
+        with patch("kvmagent.plugins.network_plugin.bash_ro", return_value=(0, lldp_json)):
+            req = _make_req({'physicalInterfaceName': 'eth0'})
+            result = plugin.get_lldp_info(req)
+        rsp = _load_rsp(result)
+
+        assert rsp['success'] is True
+        cast(MagicMock, getattr(plugin, "_get_interface_lldp")).assert_called_once_with('eth0')
```

Ruff (0.15.2)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)